src/Core/functions.php

```diff
@@ -22,6 +22,10 @@
 
 }
 
+if (!defined('CLI')) {
+    define('CLI', PHP_SAPI === 'cli');
```

Comment:
We used to have a constant like this and we removed it as there wasn't much point given that `PHP_SAPI` and `php_sapi_name()` exist.